docs/user-guide/concepts/agents/structured-output.md

```diff
@@ -52,7 +52,13 @@ class WeatherForecast(BaseModel):
     forecast_days: List[str] = Field(default_factory=list, description="Multi-day forecast")
 ```
 
-Then use the `Agent.structured_output()` method:
+Then use the `Agent.structured_output()` method as shown in the following sections.
+
+> #### Note on Usage
```

Comment:
consider using the following format for the note instead:
```
!!! note "Note on Usage"
    text here...
```
This will render it with a note background and icon like we elsewhere. For example: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#changing-the-title